### PR TITLE
Fixes a bunch of broken/redirected links

### PR DESCRIPTION
**Pull request by teoli2003**
Used the flaw dashboard to detect broken/redirected links.

### CODE REVIEW

File: files/en-us/web/api/window/showmodaldialog/index.html [hunk 1]
```diff
@@ -87,7 +87,7 @@ <h4>Note</h4>
 
 <h2 id="Examples">Examples</h2>
 
-<p><a href="/samples/domref/showModalDialog.html">Try out
+<p><a href="https://media.prod.mdn.mozit.cloud/samples/domref/showModalDialog.html">Try out
```
Comment (wbamberg) [inline files/en-us/web/api/window/showmodaldialog/index.html:90]: I think that @peterbe would like us to stop using /samples, and this API is no longer supported, so why not just remove this Examples section?

**Review by wbamberg — CHANGES_REQUESTED** [review]
Just one comment, let's remove the example for `showModalDialog`, since it doesn't work anyway.

File: files/en-us/web/api/window/showmodaldialog/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/window/showmodaldialog/index.html]: Oh! It is indeed broken. I only checked the page existed and didn't try to use.

I remove the whole `<h2>` then.

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Pull request merged**